test/parallel/test-trace-events-async-hooks-worker.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const common = require('../common');
+if (!process.binding('config').hasTracing)
+  common.skip('missing trace events');
+
+const assert = require('assert');
+const cp = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const code =
+  'setTimeout(() => { for (var i = 0; i < 100000; i++) { "test" + i } }, 1)';
+const worker = `const { Worker } = require('worker_threads');
+  const worker = new Worker('${code}',
+  { eval: true, stdout: true, stderr: true });
+  worker.stdout.on('data',
+    (chunk) => console.log('worker', chunk.toString()));
+  worker.stderr.on('data',
+    (chunk) => console.error('worker', chunk.toString()));`;
+
+const tmpdir = require('../common/tmpdir');
+const filename = path.join(tmpdir.path, 'node_trace.1.log');
+
+tmpdir.refresh();
+const proc = cp.spawnSync(
+  process.execPath,
+  [ '--trace-event-categories', 'node.async_hooks', '-e', worker ],
+  { cwd: tmpdir.path, env: {
+    'NODE_DEBUG_NATIVE': 'tracing',
+    'NODE_DEBUG': 'tracing'
+  }
```

Comment:
This will fail on some platforms (e.g. Linuxone) in the CI where `LD_LIBRARY_PATH` is used to point to the C++ runtime libraries. The tests should extend `env` rather than replace it.